modules/core/ui/theme/css/comment.css

```diff
@@ -0,0 +1,30 @@
+/*
+  Improve styling of comments. This includes the comment on
+  the entity and comment reply page.
+ */
+article.comment {
+  background: var(--gin-bg-layer);
+  border: 1px solid var(--gin-border-color-layer);
+  border-radius: var(--gin-border-m);
+  padding-left: 1em;
+  padding-inline-end: 2rem;
+  padding-bottom: 0.5em;
+  margin-top: var(--space-s);
+  margin-bottom: var(--space-s);
+}
+article.comment p.comment__submitted {
+  font-size: var(--gin-font-size-s);
+  color: var(--gin-color-text-light);
+}
+article.comment ul.links.inline {
+  margin-inline: 0;
+}
+/* Alter border style when comment is rendered on entity page. */
+/* TODO: choose a border style. */
+section.comment-wrapper article.comment {
+  /* Wrap comments in border. */
+  border: 1px solid var(--gin-border-color);
+  border-radius: var(--gin-border-m);
+  /* OR add border as left bar. */
+  border-left: 5px solid var(--gin-border-color-secondary);
+}
```

Comment:
> Actually just realized that what I'm seeing looks like a combination of both the options you described. But I like it!



> > I love this! And agree the second one is my preference too.
> 
> Actually just realized that what I'm seeing looks like a combination of both the options you described. But I like it!
> 
> ![Screenshot from 2024-03-19 10-53-36](https://private-user-images.githubusercontent.com/95381/314110314-376e2584-88ff-4d7f-adb6-309437c398a1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA4NjUzNzUsIm5iZiI6MTcxMDg2NTA3NSwicGF0aCI6Ii85NTM4MS8zMTQxMTAzMTQtMzc2ZTI1ODQtODhmZi00ZDdmLWFkYjYtMzA5NDM3YzM5OGExLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzE5VDE2MTc1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTlkZDIzN2M4NDVhOThmYTM0YjRiODE5ODFlMjM4MjRjYjExNjMyYTVlZGIxMzI3OTdjYmIxMTVkMWQ5NWRkOWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.QHWomK329tqIIDut5oPsKGGFOw0pHBeNA8V8VsAcFso)

Ha... so actually I had a TODO to choose one or the other. I included both approaches in my  WIP commit for simplicity.

I think I still prefer the second option with only the left border. It's a bit simpler but serves the purpose of highlighting each comment in the threaded hierarchy quite well.

Wrapping each comment with a full border seems a bit unnecessary and accentuates the random whitespace to the right of the comment. It might make more sense if we could wrap an entire thread in a border and make the thread collapsible. But with the default structure of the HTML this would be hard to accomplish.

Reply:
> Ha... so actually I had a TODO to choose one or the other. I included both approaches in my WIP commit for simplicity.

That explains it. :-)

I'm OK either way. Do you want to just amend your commit to use the second approach and force push?

Reply:
Done. I also decreased the space a bit around the comments and cleaned up how the comment renders when on the "reply" page.
![comment-spacing-after](https://github.com/farmOS/farmOS/assets/3116995/276b7de5-66e5-4795-9b19-b5e809e7e285)